lib/cldr-plurals/javascript_emitter.rb

```diff
@@ -84,6 +82,11 @@ def emit_value_set(value_set, operand, operator)
         end
       end
 
+      def emit_range_check(range, operand)
+        # a..b represents all *integers* between a and b, inclusive.
+        (range.start..range.finish).to_a.map { |v| "(#{emit(operand)} == #{v})" }.join(" || ")
```

Comment:
Yes, that last one looks like a winner. Floor works fine since `n >= 0` (otherwise we'd want a truncate). I prefer this for the Ruby version as well.